assets/app/view/create_game.rb

```diff
@@ -494,33 +465,37 @@ def filtered_rule?(rule)
     end
 
     def update_inputs(title_change: false)
+      return unless selected_game_or_variant
+
       title = selected_game_or_variant.title
       max_p = @max_p[title]
       min_p = @min_p[title]
-      max_players_elm = Native(@inputs[:max_players]).elm
-      min_players_elm = Native(@inputs[:min_players]).elm
+      if selected_game_or_variant
+        max_players_elm = Native(@inputs[:max_players])&.elm
+        min_players_elm = Native(@inputs[:min_players])&.elm
+      end
 
       if title_change
         max_players = max_p
         min_players = min_p
-      else
+      elsif selected_game_or_variant
         # Letters resolve to 0 when converted to integers
-        max_players = max_players_elm.value.to_i.zero? ? nil : max_players_elm.value.to_i
-        min_players = min_players_elm.value.to_i.zero? ? nil : min_players_elm.value.to_i
+        max_players = max_players_elm.value.to_i.zero? ? nil : max_players_elm.value.to_i if max_players_elm
+        min_players = min_players_elm.value.to_i.zero? ? nil : min_players_elm.value.to_i if min_players_elm
```

Comment:
max_players_elm and min_players_elm will always exist if that if on line 473 can be removed

Reply:
Also the &.value below (492 and 498)

Reply:
If a game is selected in multiplayer/hotseat, then the mode is changed to "Import hotseat game", when switching the mode back to multiplayer or hotseat, `min_players_elm` and `max_players_elm` are both `nil`.

They're also `nil` when the new game page is first loaded, and no game is yet selected; when the first game is selected, without the `&.value` on 492 and 498 those lines error out. This is new behavior since on the current `master` there is always a game selected.

I pushed up a little bit nicer version of 483 and 484 in 59ce2b9a3